sources/core/Stride.Core.MicroThreading/SchedulerEntry.cs

```diff
@@ -8,32 +8,14 @@ namespace Stride.Core.MicroThreading;
 /// <summary>
 /// Either a microthread or an action with priority.
 /// </summary>
-internal struct SchedulerEntry : IComparable<SchedulerEntry>
+internal class SchedulerEntry
```

Comment:
I haven't tested this, but just out of curiosity:
How does this change from `struct` to `class` fare from the garbage/GC standpoint? In your extreme sample scene with many scripts, doesn't this result in thousands of `SchedulerEntry` instances?